lib/store.js

```diff
@@ -110,34 +159,57 @@ class Store {
     }
     if (this.ws) {
       this._sendNextPlayerToBrowser();
-      this._sendNextPlayerTeamToBrowser();
     }
   }
-  setUserGridChoice(choice) {
-    this.grid[choice] = this.nextPlayer;
-    this.choices.push(choice);
-  }
-  async setCpuGridChoice(choice) {
-    this.grid[choice] = this.getCpuPlayer();
+  async setGridChoice(choice) {
+    const thisPlayer = this.nextPlayer;
+    const cpuPlayer = this.getCpuPlayer();
+    this.grid[choice] = thisPlayer;
     this.choices.push(choice);
-    if (this.ws) {
-      let cpuChoiceObj = {
-        type: "cpuChoice",
-        choice: choice,
-        team: this.player2.team,
-      };
-      // TODO : Await for callback
-      await sleep(500);
-      this.ws.send(JSON.stringify(cpuChoiceObj));
+    if (thisPlayer === cpuPlayer) {
+      if (this.ws) {
+        let cpuChoiceObj = {
+          type: "cpuChoice",
+          choice: choice,
+          team: cpuPlayer,
+        };
+        // TODO : Await for callback
```

Comment:
I guess we can delete this todo, given that ws.send isn't going to be nice to us.

If it becomes a problem, I suppose we could make the browser code auto-acknowledge every websocket message, and make a helper function on the backend which calls ws.send() and then waits for acknowledgement (with some timeout), but I don't think we *need* it yet.